index.html

```diff
@@ -1,22 +1,83 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
     <!-- Required meta tags -->
-    <meta charset="utf-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <meta charset="utf-8" />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1, shrink-to-fit=no"
+    />
 
     <!-- Bootstrap CSS -->
-    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/css/bootstrap.min.css" integrity="sha384-JcKb8q3iqJ61gNV9KGb8thSsNjpSL0n8PARn9HuZOnIxN0hoP+VmmDGMN5t9UJ0Z" crossorigin="anonymous">
+    <link
+      rel="stylesheet"
+      href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/css/bootstrap.min.css"
+      integrity="sha384-JcKb8q3iqJ61gNV9KGb8thSsNjpSL0n8PARn9HuZOnIxN0hoP+VmmDGMN5t9UJ0Z"
+      crossorigin="anonymous"
+    />
 
     <title>Hello, world!</title>
+    <link rel="stylesheet" href="css/styles.css" />
   </head>
   <body>
-    <h1>Hello, world!</h1>
-
+    <h1 class="text-center">Hello, world!</h1>
+    <!-- (nav.container-fluid>.row>.col-md>(ul.row.justify-content-center>(li{Home}+li{About}+li{Contact})))*1 -->
+    <nav class="container-fluid">
+      <div class="row">
+        <div class="col-md">
+          <ul class="row justify-content-center">
+            <li>Home</li>
+            <li>About</li>
+            <li>Contact</li>
+          </ul>
+        </div>
+      </div>
+    </nav>
+    <!-- (.container-sm>.row.height_80>(.col.color_purple>p{I'm left})+(.col.color_grey>p{I'm right}))*1 -->
+    <div class="container-sm">
+      <div class="row height_80">
+        <div class="col color_purple">
+          <div class="row">
+            <div class="col">
+              <span>p: (559) 800 - 9009</span>
+            </div>
+          </div>
```

Comment:
Hey, this is nice but maybe next time we could use a `p` tag to do the blocking instead of bootstrap in this case.